mobile/src/main/java/com/example/android/uamp/MediaNotificationManager.java

```diff
@@ -389,4 +403,17 @@ public void onFetched(String artUrl, Bitmap bitmap, Bitmap icon) {
             }
         });
     }
+
+    /**
+     * create Notification Channel, this is required by SDK Oreo to display notifications.
+     * */
+    @RequiresApi(Build.VERSION_CODES.O)
+    private void createNotificationChannel() {
+        NotificationChannel notificationChannel = new NotificationChannel(CHANNEL_ID, CHANNEL_NAME, NotificationManager.IMPORTANCE_LOW);
```

Comment:
Please do the following check before creating the channel:

> ```if (mNotificationManager.getNotificationChannel(CHANNEL_ID) == null) { ... }```

Please limit line length to 100 characters.